sdk/communication/AzureCommunicationChat/Tests/Recordings/test_StopPushNotifications_ReturnsSuccess.json

```diff
@@ -0,0 +1,76 @@
+{
+  "name" : "test_StopPushNotifications_ReturnsSuccess",
+  "interactions" : [
+    {
+      "recorded_at" : 1652285387.6312408,
+      "request" : {
+        "body" : {
+          "clientDescription" : {
+            "templateKey" : "AcsIos.AcsNotify_Chat_3.0",
+            "aesKey" : "0000000000000000B00000000000000000000000AES=",
+            "platform" : "iOS",
+            "platformUIVersion" : "3619\/0.0.0.0\/",
+            "appId" : "AcsIos",
+            "authKey" : "0000000000000000B0000000000000000000000AUTH=",
+            "languageId" : "",
+            "cryptoMethod" : "0x70"
+          },
+          "transports" : {
+            "APNS" : [
+              {
+                "ttl" : 15552000,
+                "path" : "mockDeviceToken",
+                "context" : ""
+              }
+            ]
+          },
+          "registrationId" : "0E0F0BE0-0000-00C0-B000-A00A00E00BD0",
+          "nodeId" : ""
+        },
+        "url" : "https:\/\/edge.skype.com\/registrar\/prod\/v2\/registrations",
+        "method" : "POST",
+        "headers" : {
+          "Content-Type" : "application\/json",
+          "User-Agent" : "azsdk-ios-AzureCommunicationChatTests\/1.0 (MacBookPro16,1 - 21.4.0; xctest:13.3 -> iOS 9.0; en_US)"
+        }
+      },
+      "response" : {
+        "status" : 202,
+        "body" : "",
+        "headers" : {
+          "x-cache" : "CONFIG_NOCACHE",
+          "x-content-type-options" : "nosniff",
+          "x-msedge-ref" : "Ref A: EC1BBE12B3F6464F9E21551EE8D80942 Ref B: YVR311000109027 Ref C: 2022-05-11T16:09:47Z",
+          "Content-Length" : "0",
+          "ms-cv" : "KR0UPiEjj0mS16SsrzrGzQ.0",
```

Comment:
What's this? Does it need to be scrubbed?

Reply:
These are included in the http response from registrar service.

Reply:
Same as above.

Reply:
Same as above.

Reply:
Verified to be system metadata.